### PR TITLE
Frontdoor RP v05-01 to use WAF from RP 04-01

### PR DESCRIPTION
**Pull request by MoustafaEid**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/frontdoor/resource-manager/readme.md [hunk 1]
```diff
@@ -272,7 +272,6 @@ input-file:
   - $(this-folder)/Microsoft.Network/stable/2020-05-01/network.json
   - $(this-folder)/Microsoft.Network/stable/2019-11-01/networkexperiment.json
   - $(this-folder)/Microsoft.Network/stable/2020-05-01/frontdoor.json
-  - $(this-folder)/Microsoft.Network/stable/2020-05-01/webapplicationfirewall.json
```
Comment (eschwabe) [inline specification/frontdoor/resource-manager/readme.md:275]: This file can be removed from the directory as well.

**Comment by MoustafaEid** [comment]
This change removes the support to link WAF policy. This feature isn't currently used, and we found that the NRP manifest hasn't been updated in production which hinders the generation for the SDK. We'll introduce this functionality again in the future once all the parts are ready.

**Comment by MoustafaEid** [comment]
JAVA SDk check keeps timing out -- this is a known issue.

**Comment by pilor** [comment]
If these APIs were never in your manifest then there is nothing to review from ARM side here.  If they were please ensure you are following the deprecation process:  https://armwiki.azurewebsites.net/api_contracts/APIDeprecationPolicy.html

**Comment by MoustafaEid** [comment]
@pilor Thanks for looking into this. The manifest did not have the related parts -- does this mean we just need the SDK's team approval?

**Comment by pilor** [comment]
> @pilor Thanks for looking into this. The manifest did not have the related parts -- does this mean we just need the SDK's team approval?

Correct

**Pull request merged**